packages/astro/src/core/errors/errors-data.ts

```diff
@@ -483,6 +483,38 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 			)} are supported for optimization.`,
 		hint: "If you do not need optimization, using an `img` tag directly instead of the `Image` component might be what you're looking for.",
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [`getStaticPaths()`](https://docs.astro.build/en/reference/api-reference/#getstaticpaths)
+	 * - [`params`](https://docs.astro.build/en/reference/api-reference/#params)
+	 * @description
```

Comment:
This description doesn't really follow our standard format for error messages. Erika has written a helpful guide here so you can get a feel for the style we're aiming for: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/README.md

This feels more like teaching someone *about* what's happening vs being user-focused by saying what went wrong and what the user needs to do next to resolve the error.

Please take a look at the error writing guide and try following that model. I'll take another look, and be happy to help, when this is a bit more user-focused!  🙌